src/XML/src/FMI3/fmi3_xml_variable.c

```diff
@@ -1230,9 +1246,32 @@ static int fmi3_xml_variable_process_attr_previous(fmi3_xml_parser_context_t* co
     {
         return -1;
     }
+
     /* Store the VR since we cannot access the variable until after parsing all variables. */
     variable->previous.vr = previous;
     variable->hasPrevious = true;
+
+    if (!fmi3_xml_variable_is_clocked(variable)) {
+        // TODO: This should not be a fatal error
```

Comment:
Setting these as fatal until we have a plan forward on treating variable parsing errors. The major issue appears that type information will not be parsed/set correctly, which then leads to segfaults.